godtools/ViewControllers/Tract/TractViewControllerContent.swift

```diff
@@ -116,4 +123,40 @@ extension TractViewController {
         snapshotView?.removeFromSuperview()
     }
     
+    private func showErrorMessage() {
+        let alert = UIAlertController(title: "error".localized,
+                                      message: "tract_loading_error_message".localized,
+                                      preferredStyle: .alert)
+        
+        let actionYes = UIAlertAction(title: "yes".localized,
+                                      style: .default,
+                                      handler: { action in
+                                        self.redownloadResources()
+        })
+        
+        let actionNo = UIAlertAction(title: "no".localized,
+                                      style: .cancel,
+                                      handler: { action in
+                                        self.disableResource()
+        })
+        
+        alert.addAction(actionYes)
+        alert.addAction(actionNo)
+        present(alert, animated: true, completion: nil)
+    }
+    
+    private func redownloadResources() {
+        DownloadedResourceManager().delete(resource!)
```

Comment:
Moving back to home happens already on this line: https://github.com/CruGlobal/godtools-swift/pull/76/files#diff-05386bdf576325a006af7b088c952098R22

This method is triggered if the user chooses the "yes" action on the error alert prompt.